File: Week 1 The MARIO problem/Im_the_tea_drinker.c
#include <stdio.h>


int main(void)
{
    int height; //get user input
    do
    {
        printf("Height: ");
        scanf("%d", &height);
    }
    while (height < 1 || height > 8);

    for (int line = 0; line < height; line++)
    {

        for (int space = height - 1; space > line; space = space - 1)   //prints spaces
        {
            printf(" ");
        }

        for (int hash = - 1; hash < line; hash++)   //prints hashes
        {

            printf("#");
        }

        printf("\n"); //new line
    }
}

File: Week 1 The MARIO problem/Im_the_tea_drinker.py
height = 0

while height < 1 or height > 8:
    height = int(input("Height: "))

for i in range(1, height + 1):
    for space in range(height - i):
        print(" ", end="")
    for hash in range(i):
        print("#", end="")
    print("  ", end="")
    for hash in range(i):
        print("#", end="")
    print()

    

